src/Tools/FirstRunCertGenerator/src/CertificateGenerator.cs

```diff
@@ -9,7 +9,7 @@ public static void GenerateAspNetHttpsCertificate()
         {
             var manager = new CertificateManager();
             var now = DateTimeOffset.Now;
-            manager.EnsureAspNetCoreHttpsDevelopmentCertificate(now, now.AddYears(1));
+            manager.EnsureAspNetCoreHttpsDevelopmentCertificate(now, now.AddYears(1), isInteractive: false);
```

Comment:
This is the bit that goes into the CLI. 

At this point we don't want to try and setup the certificate key as that requires user interaction and the CLI first run experience must remain non-interactive.

Reply:
After this change, at which point would the dev certificate first be added/updated if not during the first run experience?

Reply:
The certificate still gets created during the first run experience. We simply don't try and make it available across security partitions at that time as that's the bit that requires user interaction.